### ISSUE TITLE
SPMC 17.4 and IVUE Tv Guide

### ISSUE THREAD

**Issue by Jattah** [koying/SPMC#1021]
Trying to install Ivue guide on SPMC. All installs fairly easily, but when luanched I continue to get error downloading Default Skin. I have uninstalled and installed numeorus times. I have tried installing from a usb but still have the same issue. I tried it on Kodi 17.4 and it works fine. Anyone else having this problem. Also Ivue site has a fix for kodi where you can put in url for direct download of skin but when entered it does not work. It appears the skins folder is missing where this file should be. Any help would be appreciated

**Comment by Biscut10** [comment]
I ran into the same general issue on my Nvidia Shield (I don't know how to upload logs from there to here, and not a lot of other people seem to use or care about that specific stuff anyway) with pretty much every EPG guide I tried after the 4th beta update broke my main one. They certainly aren't playing well together. 

Work around I did is to put the IPTV stuff on 16.7.1, which I'm hoping still supports program addon EPGs going forward after the new changes go in, and simply use a separate Krypton setup for everything else if you prefer that.

**Comment by unlocKing** [comment]
Have you tried Assassins TV Guide:

https://github.com/Colossal1/repository.colossus/tree/master/script.tvguide.Assassins

**Comment by Biscut10** [comment]
No, but reading up on it now I'm guessing it'll still run into the same post-install error as the others when it comes to parsing xmltv.

Plan on giving it a try when I get home though. Thanks for pointing it out.

**Comment by Jattah** [comment]
I will also try it later, the storms passed,  sun is out in Houston, water is gone away am gonna get out, cabin fever like a MF.🤣🤣😋🤣
